handlers/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
I dunno whether `handlers` should be it's own project or if it should just be a module within the `server` project — feels a little _too_ split up now, and I suspect the code-smell is the fact they share all the dependencies.

Reply:
yup, agree, the main reason for this was to be able to add tests in the documentation, happy to move it back to `server`

Reply:
We discussed to leave this till later because we want to see what the documentation looks like with error handling first. The reason why this has been split out is because we can't do doctests in the `server` project which is building a `bin` — so if we need `doctests` because of the error handling, it may make plenty of sense to leave them split-up. 👍